app/[lang]/download/page.tsx

```diff
@@ -49,14 +50,40 @@ export default async function DownloadPage({params: {lang}, searchParams}: Downl
     day: 'numeric'
   }) || '';
 
-  const table = (items: AssetInfo[]) => {
+  const tabContent = (items: AssetInfo[], currentTab: string) => {
     if (!release) {
       return <></>
     }
 
+    const linuxPPA = currentTab === 'linux' ? <>
+      <h3 className={'text-lg font-medium mb-2'}>{dict.downloadPage.downloadPPA}</h3>
+      <CodeBlock copyString={`sudo add-apt-repository ppa:floorp/ppa && sudo apt update && sudo apt install floorp`}>
+        {`# Add the Floorp PPA on Ubuntu
+sudo add-apt-repository ppa:floorp/ppa
```

Comment:
Floorp is not hosted on Launchpad and is not available via `add-apt-repository` command.
```suggestion
curl -fsSL https://ppa.ablaze.one/KEY.gpg | sudo gpg --dearmor -o /usr/share/keyrings/Floorp.gpg
sudo curl -sS --compressed -o /etc/apt/sources.list.d/Floorp.list 'https://ppa.ablaze.one/Floorp.list'
```